gigaword-indexer/src/main/java/edu/isi/vista/gigawordIndexer/GigawordIndexer.java

```diff
@@ -0,0 +1,214 @@
+    		= new UsernamePasswordAuthenticationToken("admin", "admin");
+    	Authentication auth = authManager.authenticate(authReq);
+    	SecurityContextHolder.getContext().setAuthentication(auth);
+
+    	// Test current user is admin
+    	User user = userRepository.getCurrentUser();
+    	System.out.println("Current user: " + user.getUsername());
+    	
+    	// List projects
+    	System.out.println("List Projects: ");
+    	List<Project> projects = projectService.listAccessibleProjects(user);
+    	for (Project project : projects) {
+          System.out.println("	" + project.getName());   
+    	}
+    		
+        
+    	// create and delete a project
+    	try {
+    		
+    		// delete test project if already exists
+    		if (projectService.existsProject(PROJECT_NAME_GW)) {
+    			Project projectGigawords = projectService.getProject(PROJECT_NAME_GW);
+            	projectService.removeProject(projectGigawords);
+            }
+    		
+    		// create a project
+            System.out.println("Creating project " + PROJECT_NAME_GW);
+    		Project project = new Project();
+            project.setName(PROJECT_NAME_GW);
+            project.setMode(WebAnnoConst.PROJECT_TYPE_ANNOTATION);
+            projectService.createProject(project);
+            annotationSchemaService.initializeProject(project);
+            
+            // create a document
+            SourceDocument sourceDocument = new SourceDocument();
+            sourceDocument.setName("Gigaword Doc 1");
+            sourceDocument.setProject(project);
+            sourceDocument.setFormat("text");
+
+            String fileContent = "The capital of Galicia is Santiago de Compostela.";
+
+            // upload document
+            InputStream fileStream = new ByteArrayInputStream(
+            		fileContent.getBytes(StandardCharsets.UTF_8));
+
+            documentService.uploadSourceDocument(fileStream, sourceDocument);
+            
+            // check project and document created
+            if (projectService.existsProject(PROJECT_NAME_GW)) {
+            	Project projectGigawords = projectService.getProject(PROJECT_NAME_GW);
+            	System.out.println("Project " + projectGigawords.getName() + " created on " + projectGigawords.getCreated());
+            
+            	System.out.println("List documents under project " + projectGigawords);
+            	List<SourceDocument> documents = documentService.listSourceDocuments(projectGigawords);
+            	Iterator<SourceDocument> itr = documents.iterator();
+            	while(itr.hasNext()) {
+            		SourceDocument doc = itr.next();
+            		System.out.println("  " + doc.getName());
+            	}
+            	
+            	// delete project
+                projectService.removeProject(projectGigawords);
+            }
+            else 
+            	System.out.println("Project " + PROJECT_NAME_GW + " not created.");
+
+            
+		} catch (IOException e) {
```

Comment:
Weird spacing. Also we probably want to crash on these exceptions, rather than suppressing them